components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/util/IdentityUtil.java

```diff
@@ -1873,4 +1873,20 @@ public static boolean isSCIM2UserMaxItemsPerPageEnabled() {
         }
         return Boolean.parseBoolean(scim2UserMaxItemsPerPageEnabledProperty);
     }
+
+    /**
+     * Read the SCIM User Endpoint Consider Server Wide config and returns it.
+     *
+     * @return If SCIM User Endpoint Consider Server Wise Config is enabled.
```

Comment:
```suggestion
     * @return whether the `ConsiderServerWideUserEndpointMaxLimitEnabled` config for the SCIM User Endpoint  is enabled.
```

Reply:
I have followed the same return comment as previous methods